docs/components/HomeFeatures.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div
+    class="
+      d-container-content
+      py-16
+      sm:pt-20
+      sm:pb-24
+      relative
+    "
+  >
+    <div
+      class="
+        grid grid-cols-1
+        gap-x-6 gap-y-12
+        sm:grid-cols-2
+        lg:grid-cols-3
+        lg:gap-x-8 lg:gap-y-16
+      "
+    >
+      <HomeFeature
+        v-for="(feature, index) of features"
+        :key="index"
+        :feature="feature"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from '@nuxtjs/composition-api'
+
+export default defineComponent({
+  setup() {
+    const features = [
+      {
+        title: 'Resize and transform',
+        description:
+          'Generate images at the right size, defined in your components or following your breakpoints. Change the format and quality of your images.',
+        icon: 'IconResize'
+      },
+      {
+        title: 'Image providers',
+        description:
+          'Use your favorite image providers with ease or use the built-in one. Imgix, Cloudinary, Prismic, Twicpics, Vercel and more are supported by default.',
```

Comment:
We should avoid marketing vendor names.

Reply:
```suggestion
          'Choose from 10+ ready to use providers or use the built-in optimizer. ',
```

Reply:
Why not but we need something longer to be balanced with other features.

<img width="1440" alt="Screenshot 2021-05-27 at 18 44 16@2x" src="https://user-images.githubusercontent.com/904724/119865053-8ddc9d80-bf1b-11eb-91dc-0316d0a24efc.png">

Reply:
Since is not fair to others nuxt is agnostic 😸  Updated suggestion to be longer.

Reply:
What other sentence can we add @pi0 @danielroe?

Reply:
```suggestion
          'Choose from 10+ ready-to-use popular providers, add your own custom provider in a few lines of code - or use the default settings to generate automatically optimized static images.',
```